### PR TITLE
Radio button cleanup

### PR DESCRIPTION
**Pull request by stredger**

### CODE REVIEW

File: packages/riipen-ui/src/components/RadioButton.jsx [hunk 1]
```diff
@@ -39,10 +39,7 @@ const RadioButton = ({
         size
       )}
     >
-      <label
-        className={clsx(checked && "checked", disabled && "disabled")}
-        htmlFor={other.id || other.name}
```
Comment (ksmort) [inline packages/riipen-ui/src/components/RadioButton.jsx:44]: Do we not need htmlFor?

File: packages/riipen-ui/src/components/RadioButtonGroup.jsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const RadioButtonGroup = ({
   };
 
   const childrenWithProps = React.Children.map(
-    children,
+    children || [],
```
Comment (ksmort) [inline packages/riipen-ui/src/components/RadioButtonGroup.jsx:64]: Looks like children is a required prop, so if we want to do this (also maybe as a default prop?), should update the prop type

File: packages/riipen-ui/src/components/RadioGroup.jsx [hunk 3]
```diff
@@ -7,104 +7,118 @@ import withClasses from "../utils/withClasses";
...
-    /**
-     * A warning to display below the input.
-     */
-    warning: PropTypes.node
-  };
-
-  handleChange = e => {
-    const { onChange } = this.props;
-
+const RadioGroup = ({
```
Comment (ksmort) [inline packages/riipen-ui/src/components/RadioGroup.jsx:10]: 🙌

**Review by ksmort — CHANGES_REQUESTED** [review]
Overall lgtm!  Couple nitpicks

File: packages/riipen-ui/src/components/RadioButton.jsx (hunk 1 above)
Comment (stredger) [reply packages/riipen-ui/src/components/RadioButton.jsx:44]: this one wrapps the input so the label takes the child input, which is always what we want as its the text on the button

**Review by ksmort — APPROVED** [review]

**Pull request merged**